src/helpers/get-payload.ts

```diff
@@ -0,0 +1,43 @@
+// remove type imports from http for Deno compatibility
+// see https://github.com/octokit/octokit.js/issues/2075#issuecomment-817361886
+// import type { IncomingMessage } from "node:http";
+// declare module "node:http" {
+//    interface IncomingMessage {
+//      body?: string;
+//    }
+// }
+type IncomingMessage = any;
+
```

Comment:
This whole comment is actually no longer the case on modern versions of Deno.


```suggestion
import type { IncomingMessage } from "node:http";
declare module "node:http" {
    interface IncomingMessage {
      body?: string | Record<string, unknown> | undefined;
      rawBody?: Buffer | undefined;
    }
 }
```

The module augmentation though, will need to be seperated out for publishing on JSR.
I would move it with other module augmentations in `index.ts` which is only used on NPM

Reply:
Will accept this suggestion and move in following step